generators/client-2/templates/src/main/webapp/_index.html

```diff
@@ -10,71 +10,16 @@
     <%_ } _%>
     <meta name="viewport" content="width=device-width">
     <base href="/"/>
-    <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
-    <!-- build:css content/css/vendor.css -->
-    <!-- bower:css -->
-    <!-- endinject -->
     <%_ if(useSass) { _%>
     <link rel="stylesheet" href="content/css/vendor.css">
     <%_ } _%>
-    <!-- endbuild -->
-    <!-- build:css content/css/main.css -->
     <link rel="stylesheet" href="content/css/main.css">
-    <!-- endbuild -->
 </head>
 <body>
     <!--[if lt IE 9]>
         <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
     <![endif]-->
     <<%=jhiPrefix%>-main></<%=jhiPrefix%>-main>
-    <!-- inject:troubleshoot -->
```

Comment:
but we might still have issues like npm failed to run or webpack failed etc. so we need to tell user where to start